owncloud/core
# Adding system addressbook for users of this instance - a occ command …

…is supplied for synchronizing
- :white_check_mark:  requires https://github.com/owncloud/core/pull/20702

How to test:
- run./occ dav:sync-system-addressbook
- hook up any webdav or varddav client to remote.php/dav/addressbooks/system/system/system
- see one vcard for each user

``` sh
$ cadaver http://localhost:8888/remote.php/dav/
Authentication required for sabre/dav on server `localhost':
Username: admin
Password: 
dav:/remote.php/dav/> cd addressbooks 
dav:/remote.php/dav/addressbooks/> cd system 
dav:/remote.php/dav/addressbooks/system/> ls
Listing collection `/remote.php/dav/addressbooks/system/': succeeded.
Coll:   system                                 0  Jan  1  1970
dav:/remote.php/dav/addressbooks/system/> cd system 
ldav:/remote.php/dav/addressbooks/system/system/> ls
Listing collection `/remote.php/dav/addressbooks/system/system/': succeeded.
Coll:   system                                 0  Jan  1  1970
dav:/remote.php/dav/addressbooks/system/system/> cd system 
dav:/remote.php/dav/addressbooks/system/system/system/> ls
Listing collection `/remote.php/dav/addressbooks/system/system/system/': succeeded.
        Database:admin.vcf                   100  Nov 30 13:14
        Database:user01.vcf                  102  Nov 30 13:14
        Database:user02.vcf                  102  Nov 30 13:14
```
